### PR TITLE
Lua need fix

### PR DESCRIPTION
**Pull request by Dan54**
The game now loads ```StreamingAssets/LUA/Need.lua```

### CODE REVIEW

**Comment by Randshot** [comment]
When I add a simple line of code like `ModUtils.ULog("Need.lua loaded")`, Unity throws a lot of exception errors.

**Comment by Dan54** [comment]
Fixed

**Comment by WardBenjamin** [comment]
In NeedAction.cs, you used tabs instead of spaces.

**Comment by Dan54** [comment]
Fixed

File: Assets/Scripts/Models/World.cs [hunk 1]
```diff
@@ -166,7 +166,7 @@ void SetupWorld(int width, int height)
                 tiles[x, y].Room = GetOutsideRoom(); // Rooms 0 is always going to be outside, and that is our default room
             }
         }
-
+        new NeedActions ();
```
Comment (Mizipzor) [inline Assets/Scripts/Models/World.cs]: This doesnt look right. Why are you instantiating an object and then never use it=

Comment (Dan54) [inline Assets/Scripts/Models/World.cs]: @Mizipzor The NeedActions instantiator creates a static reference to itself, keeping a reference in World is unnecessary.

Comment (Mizipzor) [inline Assets/Scripts/Models/World.cs]: I think there's cleaner implementations of the singleton pattern, but that can be saved for another time. :)

**Comment by Mizipzor** [comment]
Tested. Game starts without errors.

**Pull request merged**